apps/ryal_core/lib/ryal/web/models/payment_gateways/stripe.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Ryal.PaymentGateway.Stripe do
+  @moduledoc "Relevant functions for working with the Stipe API."
+
+  alias Ryal.PaymentGatewayQuery
+
+  @stripe_api_key Map.get(Ryal.payment_gateways(), :stripe)
+  @stripe_base "https://#{@stripe_api_key}:@api.stripe.com"
+
+  @spec create(atom, Ecto.Schema.t, String.t) :: {:ok, String.t}
```

Comment:
I'm good with sticking to Ecto. However, we don't really need Phoenix so much as we need Plug. To me, Ryal will always require storage and web. It's up to the user if they want to mount the API. Ryal, however, would expect to be able to store data.

Unless, of course, we move the logic for Ryal into another dependency. But that's a longer-term goal. We need Ryal working first.